src/Query/Builder.php

```diff
@@ -189,7 +189,7 @@ public function project($columns)
     }
 
     /**
-     * Set the cursor timeout in seconds.
+     * The maximum amount of seconds to allow the query to run.
```

Comment:
The property doc block higher up in the file also needs to be updated. It currently says:

> The cursor timeout value.

Reply:
Right, fixed.